src/lib/detection_algorithms/config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "YOLO": "yolo",
+    "CNTK": "cntk"  
```

Comment:
CNTK is not an algorithm name :)
It should be "Fast-RCNN with CNTK" or something similiar